HealthcareSystem/Backend/Service/UsersAndWorkingTime/PatientService.cs

```diff
@@ -86,5 +86,17 @@ public List<Patient> ViewMaliciousPatients()
             }
             return result;
         }
+
+        public void BlockPatient(string jmbg)
+        {
+            Patient patient = GetPatientByJmbg(jmbg);
```

Comment:
So this is more of a taste thing, but I'd maybe catch the NotFoundException the Repository could throw here and replace it with BadRequest?